packages/runtime/src/runtime/screen/index.tsx

```diff
@@ -102,6 +103,7 @@ export const EnsembleScreen: React.FC<EnsembleScreenProps> = ({
       screen={screen}
     >
       <ModalWrapper>
+        <ScreenApiWrapper />
```

Comment:
This needs to block the rest of the tree from rendering until all the actions are evaluated properly otherwise onLoad might not invoke the API callback properly.